__tests__/contracts.js

```diff
@@ -1044,3 +1044,96 @@ test.each(ABIVersions)('Check deployed (ABI v%i)', async (abiVersion) => {
     expect(checked.alreadyDeployed)
         .toBeTruthy();
 });
+
+
+const SensorPackage = {
```

Comment:
No packages in code. Move to file as all other contracts

Reply:
done